android/src/toga_android/app.py

```diff
@@ -68,7 +68,7 @@ def onActivityResult(self, requestCode, resultCode, resultData):
 
     def onRequestPermissionsResult(self, requestCode, permissions, grantResults):
         print(
-            f"Toga app: onRequestPermissionsResult "
+            "Toga app: onRequestPermissionsResult "
             f"{requestCode=} {permissions=} {grantResults=}"
```

Comment:
Flake8 allows the redundant `f` prefix here, and I think it's better to keep it, because:

* It aligns the left edge of all the lines, making them easier to read. 

* It makes it clearer that they'll all be compiled into a single f string.

* It avoids potentially hitting [this MicroPython limitation](https://docs.micropython.org/en/latest/genrst/core_language.html#f-strings-don-t-support-concatenation-with-adjacent-literals-if-the-adjacent-literals-contain-braces).

Reply:
Fair enough - I'll restore the `f`.